disable-comments.php

```diff
@@ -393,7 +393,8 @@ public function settings_menu() {
 	}
 
 	public function settings_page() {
-		include dirname( __FILE__ ) . '/includes/settings-page.php';
+		include dirname( __FILE__ ) . '/includes/disable-comments-form.php';
+		include dirname( __FILE__ ) . '/includes/delete-comments-form.php';
```

Comment:
I think we need to have separate admin pages for these things, and keep the main settings page pretty much as it is now.